tests/h/presenters/annotation_json_test.py

```diff
@@ -1,208 +1,119 @@
 import datetime
-from unittest import mock
 from unittest.mock import create_autospec
 
 import pytest
+from h_matchers import Any
 from pyramid import security
-from pyramid.authorization import ACLAuthorizationPolicy
 
 from h.formatters import AnnotationFlagFormatter
 from h.presenters.annotation_json import AnnotationJSONPresenter
-from h.security.permissions import Permission
 from h.traversal import AnnotationContext
 
 
 class TestAnnotationJSONPresenter:
-    def test_asdict(self, document_asdict, groupfinder_service, links_service):
-        ann = mock.Mock(
-            id="the-id",
-            created=datetime.datetime(2016, 2, 24, 18, 3, 25, 768),
-            updated=datetime.datetime(2016, 2, 29, 10, 24, 5, 564),
-            userid="acct:luke",
-            target_uri="http://example.com",
-            text="It is magical!",
-            tags=["magic"],
-            groupid="__world__",
-            shared=True,
-            target_selectors=[{"TestSelector": "foobar"}],
-            references=["referenced-id-1", "referenced-id-2"],
-            extra={"extra-1": "foo", "extra-2": "bar"},
+    def test_asdict(self, annotation, context, DocumentJSONPresenter):
+        annotation.created = datetime.datetime(2016, 2, 24, 18, 3, 25, 768)
+        annotation.updated = datetime.datetime(2016, 2, 29, 10, 24, 5, 564)
+        annotation.references = ["referenced-id-1", "referenced-id-2"]
+        annotation.extra = {"extra-1": "foo", "extra-2": "bar"}
+        presenter = AnnotationJSONPresenter(context)
+
+        result = presenter.asdict()
+
+        assert result == Any.dict.containing(
+            {
+                "id": annotation.id,
+                "created": "2016-02-24T18:03:25.000768+00:00",
+                "updated": "2016-02-29T10:24:05.000564+00:00",
+                "user": annotation.userid,
+                "uri": annotation.target_uri,
+                "group": annotation.groupid,
+                "text": annotation.text,
+                "tags": annotation.tags,
+                "permissions": {
+                    "read": [annotation.userid],
```

Comment:
Technically we duplicate testing this below, but I thought it was better than being weird and having an `Any` here

Reply:
Yeah that's fine. A little duplication isn't the worst thing in the world